modules/swagger-codegen/src/main/resources/cpprest/api-source.mustache

```diff
@@ -221,9 +221,6 @@ pplx::task<{{#returnType}}{{{returnType}}}{{/returnType}}{{^returnType}}void{{/r
         throw ApiException(415, U("{{classname}}->{{operationId}} does not consume any supported media type"));
     }
 
-    //Set the request content type in the header.
-    headerParams[U("Content-Type")] = requestHttpContentType;
```

Comment:
I dropped this line because it doesn't make any good:

1. It prevents the `request.set_body` from setting a custom Content-Type (it simply ignores the later contentType passed to `request.set_body`, and thus, the multipart boundary marker cannot be set).
2. `request.set_body` sets the Content-Type in the ApiClient anyway.

Reply:
I have just seen issue #4370, so I will test it more thoroughly and report my findings.

Reply:
All my tests with POST endpoints work fine on my end:

* URL-encoded forms
* Multipart forms
* JSON body forms (NOTE: it is not enough to just set the Content-Type to make the form JSON-based)

Thus, I conclude that this change is fine.